index.js

```diff
@@ -457,16 +457,17 @@ module.exports = {
           return;
         }
       }
+      var skipThis = _.includes(self.excludeTypes, page.type);
       var url;
-      if (self.format === 'text') {
+      if (!skipThis && self.format === 'text') {
```

Comment:
Can I get some parens up in this? 37 years I've been doing this and I still twitch at the thought of assuming I know which operator will execute first

Reply:
Good idea.